relion/protocols/protocol_subtract.py

```diff
@@ -202,17 +348,21 @@ def _summary(self):
     
     # -------------------------- UTILS functions ------------------------------
     def _updateItem(self, particle, row):
-        self.reader.setParticleTransform(particle, row)
-        # FIXME: check if other attrs need saving
+        if self.isRelionInput:
+            # FIXME: check if other attrs need saving
+            particle._rlnRandomSubset = Integer(row.rlnRandomSubset)
+            self.reader.setParticleTransform(particle, row)
         particle._rlnImageOriginalName = String(row.rlnImageOriginalName)
-        particle._rlnRandomSubset = Integer(row.rlnRandomSubset)
-
-        newLoc = convert.relionToLocation(row.rlnImageName)
+        newFn = row.rlnImageName
```

Comment:
why newFn is introduced?

Reply:
Uhm,  I do not know what newFn is doing  here. This line is not in the code I am running, I will delete it.

Reply:
line deleted